py/picca/fitter2/chi2.py

```diff
@@ -127,7 +150,13 @@ def export(self):
             g.attrs['chi2'] = d.chi2(self.k, self.pk_lin, self.pksb_lin, self.best_fit.values)
             fit = g.create_dataset("fit", d.da.shape, dtype = "f")
             fit[...] = d.best_fit_model
-    
+
+        ## write down all attributes of the minimum
+        g = f.create_group("minimum")
```

Comment:
The group `'best fit'` is already the `minimum`. Please add these extra attributes there, otherwise it will be confusing.